core/src/test/scala/org/apache/spark/scheduler/WorkerDecommissionExtendedSuite.scala

```diff
@@ -32,17 +32,17 @@ import org.apache.spark.scheduler.cluster.StandaloneSchedulerBackend
 class WorkerDecommissionExtendedSuite extends SparkFunSuite with LocalSparkContext {
   private val conf = new org.apache.spark.SparkConf()
     .setAppName(getClass.getName)
-    .set(SPARK_MASTER, "local-cluster[20,1,512]")
+    .set(SPARK_MASTER, "local-cluster[10,1,512]")
```

Comment:
Any particular reason we need 10 or 20 executors? It's still too many compares to other tests, whose average number should be 2 or 3. cc @holdenk

Reply:
So I think this test came from the situation where we were experiencing a deadlock and we wanted to make sure we re-created the potential deadlock which happened when we decommissioned most of the executors. Now this deadlock never made it into OSS Spark, but having the test here to catch it just incase is good. I think we could catch the same deadlock with 5 executors and decommissioning 4 of them, but @dongjoon-hyun is the one who found this potential issue so I'll let him clarify :)

Reply:
Thank you for your explanation.